deployment/docker-compose.yml

```diff
@@ -3,14 +3,16 @@ services:
   postgres:
     image: "postgres"
     container_name: postgres
+    env_file:
+      - .env
```

Comment:
Is it correct that in order to use this `docker-compose.yml` now one needs to create `.env` file here? I think it's acceptable, but it needs to be mentioned in documentation (after you rebase this PR).

Reply:
Yes, we have to use `.env` for now.
What about our test server? I guess we have to add some variables

Reply:
Do you mean [staging](https://staging.edna.serokell.team/) server?  Well, we can update its config, but I think it's not necessary because the changes in this PR are backwards compatible (correct me if I am wrong).

Reply:
If staging server uses postgres in docker - then we have to path `.env` file on build stage with the same variables as we use in our config file for edna

Reply:
Staging server runs Docker images without `docker-compose` and it doesn't use our `deployment` folder at all. It runs them using `nix` somehow (I don't know how exactly). It only provides config file in the right place: https://github.com/serokell/gemini-infra/blob/62b9bd5988ba9a91e6d56c3ca4c156708b2e7697/servers/castor/edna.nix

And this approach should still work after this PR.